unstructured/cleaners/extract.py

```diff
@@ -0,0 +1,46 @@
+import re
+
+
+def _get_indexed_match(text: str, pattern: str, index: int = 0) -> re.Match:
+    if not isinstance(index, int) or index < 0:
+        raise ValueError(f"The index is {index}. Index must be a positive integer.")
```

Comment:
It looks like `index` is allowed to be zero, so shouldn't this say "Index must be a non-negative integer"?

Reply:
Indeed it should!

Reply:
Updated